api/v1beta1/disruption_types.go

```diff
@@ -420,6 +421,17 @@ func (status *DisruptionStatus) RemoveTargets(toRemoveTargetsCount int) {
 	}
 }
 
+// HasTarget returns true when a target exists in the Target List or returns false.
+func (status *DisruptionStatus) HasTarget(searchTarget string) bool {
+	for _, target := range status.Targets {
+		if strings.Compare(target, searchTarget) == 0 {
```

Comment:
> Out of curiosity, why using strings.Compare ?

I have used `strings.Compare` because it is slightly faster than operator comparaison. Source : https://www.jeremymorgan.com/tutorials/go/how-do-i-compare-strings-go/ 
But I am agree with you, the operator is more convenient. I will change this 👍